parsers/cxx/cxx_parser_variable.c

```diff
@@ -19,19 +19,38 @@
 #include "read.h"
 
 
-
+//
+// Attempt to extract variable declarations from the chain.
+// Returns TRUE if at least one variable was extracted.
+// Returns FALSE if a variable declaration could not be identified.
+//
+// Recognized variable declarations are of the following kinds: 
```

Comment:
Could you trim the whitespace(s) at the end of line.
(The view on github doesn't render it. But you may see it with `git diff`.